tls/s2n_psk.c

```diff
@@ -301,6 +301,18 @@ static int s2n_psk_set_hmac(struct s2n_psk *psk, s2n_psk_hmac psk_hmac_alg)
     return S2N_SUCCESS;
 }
 
+int s2n_external_psk_set_hmac(struct s2n_external_psk *external_psk, s2n_hmac_algorithm hmac_alg)
```

Comment:
This (and s2n_psk_set_hmac) should return S2N_RESULT, since they're not publicly accessible